src/visitors/Tag.js

```diff
@@ -111,6 +131,14 @@ function build(node: Object, context: Context): JSXElement {
 
 const TagVisitor = {
   jsx(node: Object, context: Context) {
+
+    if (getInterpolationRefs(node.name)) {
+      const expr = context.getInterpolationByRef(node.name);
+      if (expr) {
+        return t.jSXExpressionContainer(expr);
+      }
+    }  
```

Comment:
This would also allow for a flexibly combining interpolations and pug syntax:

const BaseName = (props) => pug`h1 ${props.title}`
const Name = (props) => pug`${BaseName}(title=${props.title})`